include/s3select_functions.h

```diff
@@ -550,6 +541,24 @@ struct _fn_add_to_timestamp : public base_function {
 
 };
 
+struct _fn_utcnow : public base_function {
+
+
+    bool operator()(std::vector<base_statement *> *args, variable *result)
+    {
+        int args_size = args->size();
+
+        if (args_size != 0) {
+            throw base_s3select_exception("utcnow does not expect any parameters");
+        }
+
+        boost::posix_time::ptime now_ptime(boost::posix_time::second_clock::universal_time());
```

Comment:
ptime is temporary variable , it should declare outside the scope of /operator()/
( on _fn_utcnow body )